<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QFormLayout" name="formLayout">
        <item row="1" column="0" colspan="2">
         <widget class="QListView" name="usersEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Пользователи</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QFormLayout" name="formLayout_2">
        <property name="horizontalSpacing">
         <number>7</number>
        </property>
        <item row="1" column="0" colspan="2">
         <widget class="QListView" name="groupsEdit"/>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>Группы</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="createGroupBut">
          <property name="text">
           <string>Создать</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="chatContainer" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
        <property name="spacing">
         <number>7</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="persChatBox">
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <property name="horizontalSpacing">
               <number>7</number>
              </property>
              <item row="0" column="0">
               <widget class="QTextBrowser" name="persChatEdit"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="groupChatBox">
             <layout class="QGridLayout" name="gridLayout_3">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QTabWidget" name="tabWidget">
                <property name="currentIndex">
                 <number>1</number>
                </property>
                <widget class="QWidget" name="chat">
                 <attribute name="title">
                  <string>Tab 1</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_5">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QTextBrowser" name="groupChatEdit"/>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="participants">
                 <attribute name="title">
                  <string>Tab 2</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_4">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QListView" name="groupUsers"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QPushButton" name="addParticipBut">
                    <property name="text">
                     <string>Добавить</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QPushButton" name="delParticip">
                    <property name="text">
                     <string>Удалить</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="settings">
                 <attribute name="title">
                  <string>Страница</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_6">
                  <item row="0" column="0">
                   <widget class="QPushButton" name="delGroupButton">
                    <property name="text">
                     <string>Удалить группу</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Orientation::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QSplitter" name="splitter_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <widget class="QTextEdit" name="messageEdit"/>
            <widget class="QPushButton" name="sendButton">
             <property name="text">
              <string>send</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
